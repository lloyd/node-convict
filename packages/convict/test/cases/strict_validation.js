'use strict';

exports.conf = {
  foo: {
    default: 'a',
    format: String
  }
};

exports.data = {
  bar: false
}

exports.validate = {
  allowed: 'strict'
};
