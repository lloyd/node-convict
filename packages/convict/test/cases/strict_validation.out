Validate failed because wrong value(s):
  - configuration param 'bar' not declared in the schema